js/views/mediacontrolsview.js

```diff
@@ -195,6 +195,14 @@
 			return true;
 		},
 
+		_getEnableVideoButtonTitle: function() {
+			if (!this._app.signaling || this._app.signaling.getSendVideoIfAvailable()) {
+				return t('spreed', 'Enable video (v)');
+			}
+
+			return t('spreed', 'Enable video (v) - In conversations with more than four participants enabling the video for the first time will cause a reconnection');
```

Comment:
Too technical, lets simplify it to:
```suggestion
			return t('spreed', 'Enable video (v) - Your connection will be briefly interrupted when enabling the video for the first time ');
```

Reply:
Done.